shared/config/secretmanager_v1beta1.yml

```diff
@@ -4,8 +4,3 @@
   :title: Secret Manager V1beta1
   :summary: Stores, manages, and secures access to application secrets.
   :env_prefix: SECRET_MANAGER
-:defaults:
-  :service:
-    :default_host: secretmanager.googleapis.com
-    :oauth_scopes:
-      - https://www.googleapis.com/auth/cloud-platform
```

Comment:
?

Reply:
The protos already define these values, so the configuration wasn't used/needed.